packages/graph/hash_graph/lib/graph/src/datastore/postgres/mod.rs

```diff
@@ -10,13 +10,36 @@ pub struct PostgresDatabase {
 }
 
 impl PostgresDatabase {
+    /// Crates a new `PostgresDatabase` connection.
```

Comment:
```suggestion
    /// Creates a new `PostgresDatabase` connection.
```

Also I had object instead of connection because this isn't a connection... It's an object with a pool inside it and we'll likely have more things than a Pool.

Reply:
Changed this in 3faf90db86130a162128ce54c1ea63f65803dc92